third_party/toolchains/bazel_0.26.0_rbe_windows/BUILD

```diff
@@ -38,6 +38,7 @@ cc_toolchain_suite(
         "x64_windows_msys": ":cc-compiler-x64_windows_msys",
         "x64_windows": ":cc-compiler-x64_windows",
         "armeabi-v7a": ":cc-compiler-armeabi-v7a",
+        "k8": ":_dummy_toolchain",
```

Comment:
why the changes? this file is autogenerated AFAIK so I don't feel comfortable updating it manually.
Would regenerating the BUILD help?
go/rbe-windows-user-guide